tools/dev_scripts/ankaios-start

```diff
@@ -6,8 +6,38 @@ LOG_DIR="$(readlink -f "$SCRIPT_DIR/../../logs")"
 
 mkdir -p "$LOG_DIR"
 
-echo "Staring up Ankaios server"
-RUST_LOG=debug /workspaces/ankaios/target/x86_64-unknown-linux-musl/debug/ank-server 2> "$LOG_DIR/ank-server" &
+start_with_tls_enabled() {
+    sudo "$SCRIPT_DIR/../certs/create_certs.sh"
 
-echo "Staring up Ankaios agent"
-RUST_LOG=debug /workspaces/ankaios/target/x86_64-unknown-linux-musl/debug/ank-agent --name agent_A 2> "$LOG_DIR/ank-agent" &
+    echo "Staring up Ankaios server with TLS enabled"
+    ANKSERVER_CA_PEM=.certs/ca.pem \
+    ANKSERVER_CRT_PEM=.certs/server.pem \
+    ANKSERVER_KEY_PEM=.certs/server-key.pem \
+    RUST_LOG=debug /workspaces/ankaios/target/x86_64-unknown-linux-musl/debug/ank-server 2> "$LOG_DIR/ank-server" &
+
+    echo "Staring up Ankaios agent with TLS enabled"
+    ANKAGENT_CA_PEM=.certs/ca.pem \
+    ANKAGENT_CRT_PEM=.certs/agent.pem \
+    ANKAGENT_KEY_PEM=.certs/agent-key.pem \
+    RUST_LOG=debug /workspaces/ankaios/target/x86_64-unknown-linux-musl/debug/ank-agent --name agent_A 2> "$LOG_DIR/ank-agent" &
+}
+
+start_with_tls_disabled() {
+    echo "Staring up Ankaios server with TLS disabled"
+    RUST_LOG=debug /workspaces/ankaios/target/x86_64-unknown-linux-musl/debug/ank-server --insecure 2> "$LOG_DIR/ank-server" &
+
+    echo "Staring up Ankaios agent with TLS disabled"
+    RUST_LOG=debug /workspaces/ankaios/target/x86_64-unknown-linux-musl/debug/ank-agent --insecure --name agent_A 2> "$LOG_DIR/ank-agent" &
+}
+
+if [ -z "$1" ] || [ "$1" != "insecure" ]; then
```

Comment:
I would suggest to use "--insecure" to keep it consistent with the parameter in `ank`, `ank-server` etc.
```suggestion
if [ -z "$1" ] || [ "$1" != "--insecure" ]; then
```